frontend/src/components/search/SearchBar.tsx

```diff
@@ -2,15 +2,15 @@ import React from "react";
 
 export default function SearchBar() {
   return (
-    <form className="usa-search usa-search--big" role="search">
+    <div className="usa-search usa-search--big" role="search">
       <label className="usa-sr-only" htmlFor="search-field">
         Search
       </label>
       <input
         className="usa-input maxw-none"
         id="search-field"
         type="search"
-        name="search"
+        name="search-keyword"
```

Comment:
I'm not sure if the type should change also? A generic `search-keyword` seemed more specific when referencing this input - but if there's a better name lmk